src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,42 @@
+package duke;
+
+import java.util.Scanner;
+
+import java.io.IOException;
+import java.io.File;
+
+public class Duke {
+    public static void main(String[] args) throws IOException {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        System.out.println("What can I do for you?");
+        Scanner in = new Scanner(System.in);
+        TaskList list = new TaskList();
+        File file = FileHandler.loadFile();
+        FileHandler.readFile(list,file);
+        String request = in.nextLine();
+        while (CommandType.isNotBye(request)) {
+            try {
+                if (CommandType.isList(request)) {
+                    list.printTasks();
+                } else if (CommandType.isDone(request)) {
+                    list.doneTask(request);
+                } else if (CommandType.isDelete(request)) {
+                    list.deleteTask(request);
+                } else {
+                    list.addTask(request);
+                }
```

Comment:
Consider abstracting this into your RequestParser class as this part of the code is doing some parsing/making sense of user inputs